### PR TITLE
Release of 3.4.0

### PR DESCRIPTION
**Pull request by trevor-vaughan**
#### Pull Request (PR) description

Release of 3.3.2

### CODE REVIEW

**Review by ekohl — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -157,12 +169,11 @@ These should not affect the functionality of the module.
 
 ## [v1.5.3](https://github.com/voxpupuli/puppet-selinux/tree/v1.5.3) (2018-08-31)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v1.5.2...v1.5.3)
+[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/simp-1.5.2...v1.5.3)
```
Comment (ekohl) [inline CHANGELOG.md:172]: Wait, this line is odd.

File: CHANGELOG.md [hunk 2]
```diff
@@ -171,6 +182,14 @@ These should not affect the functionality of the module.
 - Remove docker nodesets [\#257](https://github.com/voxpupuli/puppet-selinux/pull/257) ([bastelfreak](https://github.com/bastelfreak))
 - drop EOL OSs; fix puppet version range [\#256](https://github.com/voxpupuli/puppet-selinux/pull/256) ([bastelfreak](https://github.com/bastelfreak))
 
+## [simp-1.5.2](https://github.com/voxpupuli/puppet-selinux/tree/simp-1.5.2) (2018-07-13)
```
Comment (ekohl) [inline CHANGELOG.md:185]: What's this version?

**Review by ekohl — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (trevor-vaughan) [reply CHANGELOG.md:185]: Huh...weird...I'll correct that.

File: CHANGELOG.md [hunk 3]
```diff
@@ -157,12 +169,11 @@ These should not affect the functionality of the module.
 
 ## [v1.5.3](https://github.com/voxpupuli/puppet-selinux/tree/v1.5.3) (2018-08-31)
 
-[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v1.5.2...v1.5.3)
+[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/1.5.2...v1.5.3)
```
Comment (ekohl) [inline CHANGELOG.md:172]:
```suggestion
[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v1.5.2...v1.5.3)
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -171,6 +182,14 @@ These should not affect the functionality of the module.
 - Remove docker nodesets [\#257](https://github.com/voxpupuli/puppet-selinux/pull/257) ([bastelfreak](https://github.com/bastelfreak))
 - drop EOL OSs; fix puppet version range [\#256](https://github.com/voxpupuli/puppet-selinux/pull/256) ([bastelfreak](https://github.com/bastelfreak))
 
+## [1.5.2](https://github.com/voxpupuli/puppet-selinux/tree/1.5.2) (2018-07-13)
```
Comment (ekohl) [inline CHANGELOG.md:185]: We already have a 1.5.2 version. I think the only closed issue here shouldn't be moved from where it was.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (trevor-vaughan) [reply CHANGELOG.md:185]: Ah, not sure what's going on with the script :-/

Comment (trevor-vaughan) [reply CHANGELOG.md:185]: Oh, I had a leftover tag from another project in a different remote :-/. That's fancy.

Comment (trevor-vaughan) [reply CHANGELOG.md]: Sorry about that, hopefully this is fixed now. It looks like one of the rake tasks may have decided to sync my local tags

**Comment by trevor-vaughan** [comment]
@ekohl Sorry for the trouble, this should be fixed now.

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
...
+
+[Full Changelog](https://github.com/voxpupuli/puppet-selinux/compare/v3.3.1...v3.3.2)
+
+**Closed issues:**
+
+- Resources that use 'semanage' should have an autorequires on the various packages [\#345](https://github.com/voxpupuli/puppet-selinux/issues/345)
+
+**Merged pull requests:**
+
+- Add package autorequires to native types [\#346](https://github.com/voxpupuli/puppet-selinux/pull/346) ([trevor-vaughan](https://github.com/trevor-vaughan))
```
Comment (ekohl) [inline CHANGELOG.md:17]: Can we label this as an enhancement or bug fix?

**Review by ekohl — COMMENTED** [review]
Apologies for not catching this on the first review.

File: CHANGELOG.md (hunk 5 above)
Comment (trevor-vaughan) [reply CHANGELOG.md:17]: Oh, sure. I guess it's an enhancement.

**Review by ekohl — APPROVED** [review]
Thanks!

File: metadata.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-selinux",
-  "version": "3.3.2-rc0",
+  "version": "3.3.2",
```
Comment (ekohl) [inline metadata.json:3]: Typically enhancements get a minor version.

**Pull request merged**